docs/pages/access-controls/sso/okta.mdx

```diff
@@ -134,22 +144,97 @@ You can also right click on the "View IdP metadata" link and select
 
 ## Step 3/4. Create a SAML connector
 
-Define a SAML connector resource in a local file named `okta-connector.yaml`:
+Define an Okta SAML connector using `tctl`. Update this example command with
+the path to your metadata file, and edit the `--attributes-to-roles` values for
+custom group assignment to roles. See [tctl sso configure
+saml](../../reference/cli.mdx#tctl-sso-configure-saml) for a full reference of
+flags for this command:
 
-```yaml
-(!examples/resources/saml-connector.yaml!)
+```code
+$ tctl sso configure saml --name=okta \
```

Comment:
We do have `--preset` flag and `okta` is one of the allowed values. If you choose a preset, `--name` is no longer required: https://github.com/gravitational/teleport.e/blob/f7f187a26028f7144404abae5e5ef44d395bd195/tool/tctl/sso/configure/saml.go#L166-L168

In addition to setting a name, the preset can modify the settings in a particular way. Right now the only non-cosmetic change happens for `adfs`. We still set `description` and `display` values. 

See also: https://github.com/gravitational/teleport.e/blob/f7f187a26028f7144404abae5e5ef44d395bd195/tool/tctl/sso/configure/saml.go#L51-L59

Reply:
Resolved in 7565debd0b8fafc9c5ab865b7e222a63f013cca9